finance/funding_requests.md

```diff
@@ -0,0 +1,52 @@
+
+Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+
+
+## Travel
+
+1. Send an email to astropy-finance-committee@googlegroups.com with the following information:
+* Name
+* Description
+* Total Request
+* Itemized Budget
+
+Costs for per diem can use the [US Government per deim rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 
+
+2. The financd committee will review the costs and provide approval within 1 week of the request. 
+
+3.  Travel occurs
+
+4.  Preferable within two weeks of completing the trip, travel costs are submitted via the [NumFOCUS Rocket form | https://numfocus.org/rocket]
+
+5. NumFOCUS provides the information for approval to the Finance Committee.
+
+6. At least, one member of the finance committee provides approval for the payment.
+
+7. Payment is made to the requester for travel. 
+
+
+## Contractor Work
+
+1.  A request for contract work should be made via email to astropy-finance-committee@googlegroups.com.  
```

Comment:
Requests should be for a substantive amount of work (e.g., at least 30 hours/month). The request should include a description of the proposed work, relevance to project-wide goals, and suggested method of payment (e.g., subaward to institution, hourly contract).

Reply:
I generally agree with @kelle but that's exactly the point I wanted to get to in the end: How do we decide who gets paid for exactly how much work? In practice, I imagine that we might select which issues need to be worked on and have an application process or some way to work out whom of all the qualified project members to assign to that.  
But what I'm saying here is the process to be worked out - we don't have that yet and we should say so.